### DISCUSSION TITLE
Display a combination of keys in menu bar on Mac OS

### DISCUSSION THREAD

**Discussion by MrPandir** [Chatterino/chatterino2#5302] [Ideas] [answered]
Now only this is reflected there:
<img width="352" alt="Screenshot 2024-04-06 at 14 13 25" src="https://github.com/Chatterino/chatterino2/assets/137798474/306b3b05-a4ca-4e7c-9229-a72988f6b30e">

**Comment by pajlada** [comment, marked as answer]
I think this has been improved in https://github.com/Chatterino/chatterino2/pull/5266

**Reply by MrPandir** [reply]
Magnificently, I await the next stable release.
